HW03/ekaterina_balabanovich/spec/balabanovich_hw03_task_1_spec.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+require_relative '../balabanovich_hw03_task_1'
+
+RSpec.describe 'Task1' do
+  describe '#error_extract' do
+    context 'when one error occurred' do
+      let(:one_error_logs) do
```

Comment:
I don't think that it's necessary to use different naming for each log it's already clear from the context description 
```suggestion
    context 'when one error occurred' do
      let(:logs) do
```